kubernetes/backend-service.json

```diff
@@ -24,7 +24,7 @@
     "selector": {
       "io.kompose.service": "backend"
     },
-    "clusterIP": "10.110.129.92",
+    "clusterIP": "10.245.0.1",
```

Comment:
I think, that's fine! Does Digital Ocean offer this particular IP address or does it have to be within a certain range?

Maybe we can remove it and DO assigns an IP address?